plugins/powerlevel/powerlevel

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+export DOTFILES_POWERLEVEL_CONFIG=('enable')
+
+dotfiles_powerlevel_prompt_string ()  {
+  case $1 in
+    enable)
+      echo 'Do you want to use powerlevel10k to configure your prompt? (1 for yes, anything else for no)'
+      return 0
+      ;;
+    esac
+  return 1
+}
+
+dotfiles_powerlevel_apply  () {
+  debug 'checking if we want powerlevel10'
+  if [ "$(plugin_config_get ntfy pushover_key)" -ne "1" ]; then
```

Comment:
`ntfy pushover_key`?

Reply:
yea, that's one of those commits i mentioned that's still coming. i've got a bunch of changes on my other computer, but it's not done running yet (waiting for encryption stuff to finish) so once that's done, i'll update this.